### DISCUSSION TITLE
Robot emergency stop

### DISCUSSION THREAD

**Discussion by Rabibabo** [Victorlouisdg/cloth-competition#29] [Questions]
When grabbing the short side of the towel, the two robotic arms were not level. The torque detected by the joints was too large, causing the robotic arms to stop suddenly. Will this happen in competitions? How do you avoid this problem? .
In addition, there are other situations that cause the robot arm to stop suddenly, so that the code will crash. So how should the result of the game be determined? What should I do if I encounter this situation, especially after Grasp_hanging? If the simple determination is to do it again, then Would it be unfair?
![微信图片_20240510190413](https://github.com/Victorlouisdg/cloth-competition/assets/130025759/799d14e2-6df2-4e35-a63d-c0b733b0a25c)
![微信图片_20240510190418](https://github.com/Victorlouisdg/cloth-competition/assets/130025759/eb0f33d0-6581-4185-b02e-ad0db2829f1b)

**Comment by Victorlouisdg** [comment]
We understand your concerns. In the event that a protective stop is triggered during the competition, your team will have the opportunity to retry the trial from the start. We have tried to minimize such occurrences, but planning the motions of two robots holding cloth can be challenging.
